postgres/tests/test_relations.py

```diff
@@ -185,54 +185,76 @@ def test_index_metrics(aggregator, integration_check, pg_instance):
 
 @pytest.mark.integration
 @pytest.mark.usefixtures('dd_environment')
-def test_locks_metrics(aggregator, integration_check, pg_instance):
```

Comment:
@alexandre-normand refactored all the locks metrics tests into a single table test, and added cases to test the table name exclusion logic